middleware/categories.js

```diff
@@ -0,0 +1,10 @@
+import categories from '@/assets/js/categories'
+
+const categoryIds = categories.map(cat => cat.id)
```

Comment:
This is something that always makes me doubt. Here we have the ids of categories so... shouldn't it be `categoriesIds` ?

Reply:
For me, both of them are ok, since one is for the generic term of category, and the other is for specifically this array categories. Since both mean the same, it's fine with me :)